tests/unittests/test_functions_registry.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import unittest
+from azure.functions import Function
+from azure.functions.decorators.http import HttpTrigger
+
+from azure_functions_worker import functions
+from azure_functions_worker.functions import FunctionLoadError
+
+
+class TestFunctionsRegistry(unittest.TestCase):
+
+    def setUp(self) -> None:
+        def dummy():
+            return "test"
+
+        self.dummy = dummy
+        self.func = Function(self.dummy, "test.py")
+        self.function_registry = functions.Registry()
+
+    def test_add_index_functions_invalid_route(self):

```

Comment:
add a test for happy path or already exists?

Reply:
Happy paths already exists in the form on e2e tests.